neuro_san/service/grpc/grpc_agent_server.py

```diff
@@ -88,32 +87,23 @@ def __init__(self, port: int,
 
         self.network_storage_dict: Dict[str, AgentNetworkStorage] = network_storage_dict
         # Below is now odd.
-        self.agent_networks: Dict[str, AgentNetwork] = agent_networks
         self.server_name: str = server_name
         self.server_name_for_logs: str = server_name_for_logs
         self.max_concurrent_requests: int = max_concurrent_requests
         self.request_limit: int = request_limit
+        self.server_status: ServerStatus = server_status
 
         self.server_lifetime = None
         self.security_cfg = None
         self.services: List[GrpcAgentService] = []
         self.service_router: DynamicAgentRouter = DynamicAgentRouter()
-        # Event to notify that we have started serving
-        self.notify_started: threading.Event = threading.Event()
-        self.logger.info("agent_networks found: %s", str(list(self.agent_networks.keys())))
 
     def get_services(self) -> List[GrpcAgentService]:
         """
         :return: A list of the AgentServices being served up by this instance
         """
         return self.services
 
-    def get_starting_event(self) -> threading.Event:
-        """
```

Comment:
We don't need threading.Event to sync starting of gRPC and http services - they don't depend on each other.